homeassistant/components/asuswrt/router.py

```diff
@@ -408,6 +419,17 @@ def api(self) -> AsusWrt:
         return self._api
 
 
+async def _get_nvram_info(api: AsusWrt, info_type):
```

Comment:
Missing return value typing and the type of the second parameter. Please type the whole signature when adding type annotation.

Reply:
Created PR #51153